doc/hardware/porting/soc_porting.rst

```diff
@@ -128,7 +128,7 @@ Multiple SoCs and SoC series in a common folder can be described in the
 .. code-block:: yaml
 
    family:
-     name: <family-name>
+   - name: <family-name>
```

Comment:
this is actually missing an indent like the parts below applied to the parts below also, needs to be:
```
   family:
     - name: <family-name>
       series:
         - name: <series-1-name>
           socs:
             - name: <soc1>
               cpuclusters:
                 - name: <coreA>
                 - name: <coreB>
                   ...
             - name: <soc2>
         - name: <series-2-name>
           ...
```

Reply:
No, indentation for block sequences is not required, nor enforced anywhere in any of our YAML files. Only needs to be [consistent](https://github.com/zephyrproject-rtos/zephyr/blob/main/.yamllint#L12).
In fact, looking at soc.yml files added over the last 3 months, it's basically 50/50.

Please revisit.

Reply:
Ah - my bad, so that file is *not* consistent. Completely missed it, my sincere apologies. Fixing ...